Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11.11-slim
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py", "runserver", "0.0.0.0:8000"]
+
+# For run docker:   docker run -p 8001:8000 --env-file .env killmont/docker-weather-api

```

Comment:
Verify that the image name `killmont/docker-weather-api` in the comment matches the image you have built or intend to use. This will help avoid confusion when running the container.